rsconf/component/sirepo.py

```diff
@@ -194,7 +197,8 @@ def _install_dirs_and_files(self):
             )
 
     def _jupyterhublogin(self, z):
-        z.jupyterhub_enabled =  'jupyterhublogin' in self.j2_ctx.sirepo.feature_config.default_proprietary_sim_types
+        z.jupyterhub_enabled =  'jupyterhublogin' in set(self.j2_ctx.sirepo.feature_config.moderated_sim_types).union(
```

Comment:
I don't think this will work. default_proprietary_sim_types will go away eventually so the code must be robust to that. use `get`.

Reply:
87e5cf5